templates/zerver/api/get-presence.md

```diff
@@ -0,0 +1,59 @@
+# Get user presence
+
+Get the presence status for a specific user.
+
+`GET {{ api_url }}/v1/users/<email>/presence`
+
+## Usage examples
+
+<div class="code-section" markdown="1">
+<ul class="nav">
+<li data-language="python">Python</li>
+<li data-language="curl">curl</li>
+</ul>
+<div class="blocks">
+
+<div data-language="curl" markdown="1">
+
+```
+curl {{ api_url }}/v1/users/<email>/presence \
+    -u BOT_EMAIL_ADDRESS:BOT_API_KEY
+```
+
+</div>
+
+<div data-language="python" markdown="1">
+
+{generate_code_example(python)|/users/{email}/presence:get|example}
+
+</div>
+
+</div>
+
+</div>
+
+## Arguments
+
+{generate_api_arguments_table|zulip.yaml|/users/{email}/presence:get}
+
+## Response
+
+#### Return values
+
+* `presence`: An object containing the presence details for every client the
+  user has logged into.
+    * `<client_name>` or `aggregated`: the keys for these objects are the names
+      of the different clients where this user is logged in, like `website` or
+      `ZulipMobile`. There is also an `aggregated` key, which matches the
+      contents of the object that has been updated most recently.
```

Comment:
We should mention that most clients should just look at the `aggregated` field.

Also, let's open a follow-up issue for making an article on "User agents, clients, and Zulip" that will talk about things like `ZulipMobile` in more detail.  There's probably something in ReadTheDocs we can steal.

Reply:
Done, and created #10415 as well.